src/payment/offsite-payment-mappers/payload-mapper.js

```diff
@@ -87,6 +87,10 @@ export default class PayloadMapper {
             this.storeMapper.mapToStore(data)
         );
 
+        if (payment.formattedPayload) {
+            objectAssign(payload, payment.formattedPayload);
```

Comment:
should we be adding a new attribute that contains the formattedPayload?
e.g. 


```js
payload.browser_info = payment.formattedPayload
```

Reply:
As in the comment above, I used `objectAssign` to append the contents of the `formattedPayload` into the `payload` so the fields are sent separately to maintain the consistency with all the fields that are already sent by the mapper.